sdk/search/azure-search-documents/README.md

```diff
@@ -156,6 +156,8 @@ for (SearchResult result : indexClient.search("luxury hotel",
 }
 ```
 
+- Samples are explained in detail [here][samples_readme].
```

Comment:
This is already included in the `Next Steps` section, do we need add it again?

Reply:
This is the feedback from readme review meeting.

Reply:
Interesting, I believe we removed this in the past for the same reason. I'm okay with duplicating it than.